web/sites/default/settings.ts.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+/**
+ * If not on Pantheon - load our .env file.
+ */
+if (!defined('PANTHEON_ENVIRONMENT')) {
+  $dotenv = new Dotenv\Dotenv(__DIR__ . '/../../../');
+  $dotenv->overload();
+}
+/**
+ * Skipping permissions hardening will make scaffolding
+ * work better, but will also raise a warning when you
+ * install Drupal.
+ *
+ * https://www.drupal.org/project/drupal/issues/3091285
+ */
+// $settings['skip_permissions_hardening'] = TRUE;
+
+
+// If we have a default hash salt value (from PRESSFLOW_SETTINGS) use it.
+if (!empty($drupal_hash_salt)) {
+  $settings['hash_salt'] = $drupal_hash_salt;
+}
+
+$settings['config_sync_directory'] = '../config';
+
+if (defined('PANTHEON_ENVIRONMENT')) {
+  $config['config_split.config_split.local']['status'] = FALSE;
+}
+else {
+  $config['config_split.config_split.local']['status'] = TRUE;
+}
+
+// If we are not installing and have a redis host defined, setup redis.
+if (!(getenv('DRUPAL_INSTALL') || $is_installer_url) && !empty($conf['redis_client_host'])) {
```

Comment:
This reference to `$is_installer_url` is complaining for me when I run drush commands, saying that at this point in execution that var doesn't exist yet--which, as far as I understand, it doesn't, because although it has been loaded into `$_ENV` by the first chunk of code in this file, it won't be made a variable until we include settings.pantheon.php. I'm hesitant to suggest a fix here because you've done so much more work on this than I have, but let me know if you want to rubber-duck or brainstorm together.